components/utils/icon/build.gradle

```diff
@@ -1,4 +1,7 @@
 apply plugin: 'com.android.library'
+apply from: '../../../buildSrc/pmd.gradle'
+apply from: '../../../buildSrc/checkstyle.gradle'
+apply from: '../../../buildSrc/findbugs.gradle'
```

Comment:
How about replacing the relative path with rootProject path?
"$rootProject.projectDir/buildSrc/findbugs.gradle"